src/button-icons/button-icons.css

```diff
@@ -0,0 +1,64 @@
+.omrs-btn-icon-medium {
+  width: 2rem;
+  height: 2rem;
+  text-align: center;
+  background: transparent;
+  border: none;
+  font-size: 0;
```

Comment:
Why is this here? There's no text inside of the buttons, right, so why change the font size?

Reply:
Removed it. I forgot to clean a little.